jdi-light-material-ui/src/main/java/com/epam/jdi/light/material/elements/feedback/Snackbar.java

```diff
@@ -43,55 +59,69 @@ public void close() {
         }
     }
 
+    /**
+     * Checks if the snackbar message is of the specified type (e.g. error message, success message).
```

Comment:
Checks that the snackbar message is of the given type (e.g. error message, success message) or not.